interfaces/Portal/src/app/services/program-phase.service.ts

```diff
@@ -1,13 +1,10 @@
 import { Injectable } from '@angular/core';
-import { Router } from '@angular/router';
-import { AppRoutes } from '../app-routes.enum';
-import { ProgramPhase } from '../models/program.model';
-import { PROGRAM_PHASE_ORDER } from '../program-phase-order';
-import { ProgramsServiceApiService } from './programs-service-api.service';
+import { ProgramTab } from '../models/program.model';
+import { PROGRAM_TABS_ORDER } from '../program-phase-order';
 
-export class Phase {
+export class Tabs {
```

Comment:
```suggestion
export class Tab {
```

A type/class should be singular, I think.